Simplenote/Classes/SPNoteListViewController+Extensions.swift

```diff
@@ -834,6 +834,30 @@ extension SPNoteListViewController {
     }
 }
 
+// MARK: - Excerpt
+//
+private extension SPNoteListViewController {
+
+    func bodyExcerpt(from note: Note, withKeywords keywords: [String]?) -> String? {
+        guard let keywords = keywords, !keywords.isEmpty, let content = note.content else {
+            return note.bodyPreview
+        }
+
+        guard let bodyRange = NoteContentHelper.structure(of: content).body else {
+            return note.bodyPreview
```

Comment:
Q: in this case, wouldn't it be the same as doing `return nil` ?

Reply:
Yes! Changed :-)